src/mlpack/methods/preprocess/preprocess_split_main.cpp

```diff
@@ -73,6 +80,7 @@ PARAM_DOUBLE_IN("test_ratio", "Ratio of test set; if not set,"
     "the ratio defaults to 0.2", "r", 0.2);
 
 PARAM_INT_IN("seed", "Random seed (0 for std::time(NULL)).", "s", 0);
+PARAM_FLAG("shuffle_data", "Avoid shuffling and splitting the data.", "S");
```

Comment:
If this flag indicates that we *shouldn't* shuffle, should we name it something like `no_shuffle`?

Reply:
thanks for the naming, it handled @favre49 issue on this pr also :)